### ISSUE TITLE
Evercreech Junction

### ISSUE THREAD

**Issue by djw-zr** [rickloader/NewForestRouteV3#16]
I have found at problem with file Watersnake.tdb at the Everscreech Junction.  Junction nodes 8766 and 8775 have the outgoing pins (1 and 2) connected to VectorNodes, but in both cases pin '0' is connected to non-existent track section '0'.  

On investigation, the two nodes together represent a back to back junction.  As I understand it all junction and end nodes should be joined only to vector nodes - possibly of zero length.  I have got around the problem by treating the missing section as an end-of-line node - I presume that OpenRails does the same.

I checked for similar problems elsewhere and found that TrackNode 2819 is similar.  However it is a TrEndNode and is not referred to by any other nodes.

I have not tried to edit the file and submit a patch as I would have to guess the format of the correct zero length node.  I presume other contributors have the software to do this correctly

On looking around Everscreech, there is also some disconnected track made up of nodes 8705, 8706 and 8707.

Best regards,  David.

**Comment by rickloader** [comment]
Thanks David,  The 2 illegal back to  back junction nodes have been deleted. To gain space a sharper turnout inserted. the geometry not quite correct but OR should accept. Location Evercreech Jct centre siding to goods shed.
Buried extraneoust rack sections deleted. updated on github
Many thanks, Rick

**Comment by rickloader** [comment]
Thanks very much David. I really appreciate your finding this error and reporting it in a detailed way.
have deleted the 2 back to back turnouts. To gain space between nodes I have used a shorter 11d turnout. The geometry is not quite correct, but I think it is valid for OR
all the best, Rick

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

From: ***@***.***>
Sent: 17 July 2023 13:04
To: ***@***.***>
Cc: ***@***.***>
Subject: [rickloader/NewForestRouteV3] Evercreech Junction (Issue #16)


I have found at problem with file Watersnake.tdb at the Everscreech Junction. Junction nodes 8766 and 8775 have the outgoing pins (1 and 2) connected to VectorNodes, but in both cases pin '0' is connected to non-existent track section '0'.

On investigation, the two nodes together represent a back to back junction. As I understand it all junction and end nodes should be joined only to vector nodes - possibly of zero length. I have got around the problem by treating the missing section as an end-of-line node - I presume that OpenRails does the same.

I checked for similar problems elsewhere and found that TrackNode 2819 is similar. However it is a TrEndNode and is not referred to by any other nodes.

I have not tried to edit the file and submit a patch as I would have to guess the format of the correct zero length node. I presume other contributors have the software to do this correctly

On looking around Everscreech, there is also some disconnected track made up of nodes 8705, 8706 and 8707.

Best regards, David.
